public/models/CreateDetector/components/ConfigureAlerts/components/AlertCondition/AlertConditionPanel.mock.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { AlertCondition } from '../../../../../../../models/interfaces';
+import {
+  NotificationChannelOption,
+  NotificationChannelTypeOptions,
+} from '../../../../../../pages/CreateDetector/components/ConfigureAlerts/models/interfaces';
+import { CreateDetectorRulesOptions } from '../../../../../types';
+import { mockDetector, mockTriggerAction } from '../../../../../Interfaces.mock';
+import { RuleOptions, RulesPage } from '../../../../../interfaces';
+
+export const mockAlertCondition: AlertCondition = {
+  // Trigger fields
+  name: 'alertCondition',
+  id: 'alertConditionId',
+
+  // Detector types
+  types: ['windows', 'dns'],
+
+  // Trigger fields based on Rules
+  sev_levels: ['low', 'high', 'critical'],
+  tags: ['mock.tag'],
+  ids: ['ruleID1', 'ruleID2'],
+
+  // Alert related fields
+  actions: [mockTriggerAction],
+  severity: 'low',
+};
+
+export const mockNotificationChannelOption: NotificationChannelOption = {
+  label: 'notificationChanelOptionLabel',
+  value: 'notificationChanelOptionValue',
+  type: 'notificationChanelOptionType',
+  description: 'notificationChanelOptionDescription',
+};
+export const mockNotificationChannelTypeOptions: NotificationChannelTypeOptions = {
```

Comment:
Add new lines before the export statements

Reply:
Added.